scapy/config.py

```diff
@@ -345,6 +345,7 @@ class Conf(ConfClass):
     L3socket = None
     L2socket = None
     L2listen = None
+    min_pkt_size = 60
```

Comment:
Why did you choose 60 ?

According to the page linked in issue #90 this value could be lower. For example, on my laptop `sendp("A"*15)` works fine while `sendp("A"*14)` fails. _15_ is indeed equal to `len(str(Ether()))`. I wonder if it could be computed dynamically.